lib/transport/datagouvfr/client.ex

```diff
@@ -25,6 +25,15 @@ defmodule Transport.Datagouvfr.Client do
     |> Transport.Datagouvfr.Client.request(conn, url, body, headers, opts)
   end
 
+  @spec post_request(%Plug.Conn{}, binary, OAuth2Client.body,
+                    OAuth2Client.headers, Keyword.t)
+                    :: {:ok, Response.t} | {:error, Error.t}
```

Comment:
Those types exist ?

Reply:
Yep, the importation is here https://github.com/l-vincent-l/transport-site/blob/aca2e3f0c061c3873e3cac92fa56c6845ac28d7e/lib/transport/datagouvfr/client.ex#L8